packages/@react-spectrum/autocomplete/test/SearchAutocomplete.test.js

```diff
@@ -115,9 +115,13 @@ function testSearchAutocompleteOpen(searchAutocomplete, listbox, focusedItemInde
   expect(items[1]).toHaveTextContent('Two');
   expect(items[2]).toHaveTextContent('Three');
 
-  expect(listbox).not.toHaveAttribute('tabIndex');
+  expect(listbox).toHaveAttribute('tabIndex', typeof focusedItemIndex === 'undefined' ? '0' : '-1');
```

Comment:
do we expect focusedItemIndex to be undefined at this point? or is this a React 16/17/18 difference?